### ISSUE TITLE
[Bug]:  v3.0 Does not support cocoapods?

### ISSUE THREAD

**Issue by rgmyyw** [mapbox/mapbox-navigation-ios#4646]
### Mapbox Navigation SDK version

v3.0.2

### Steps to reproduce

```
  pod 'MapboxSearch', "2.0.2"
  pod 'MapboxSearchUI', "2.0.2"
  pod 'MapboxMaps'
  pod 'MapboxStatic.swift'
  
  pod 'MapboxCoreNavigation'
  pod 'MapboxNavigation'



pod install 


[!] CocoaPods could not find compatible versions for pod "MapboxCommon":
  In Podfile:
    MapboxMaps was resolved to 10.0.1, which depends on
      MapboxCommon (= 20.0.0)

    MapboxSearch (= 2.0.2) was resolved to 2.0.2, which depends on
      MapboxCommon (~> 24.4.0-beta.2)

CocoaPods could not find compatible versions for pod "MapboxCoreNavigation":
  In Podfile:
    MapboxCoreNavigation

    MapboxNavigation was resolved to 1.0.0, which depends on
      MapboxCoreNavigation (= 1.0.0)

CocoaPods could not find compatible versions for pod "MapboxMaps":
  In Podfile:
    MapboxMaps

There are only pre-release versions available satisfying the following requirements:

	'MapboxMaps', '>= 0'

You should explicitly specify the version in order to install a pre-release version
CocoaPods could not find compatible versions for pod "Polyline":
  In Podfile:
    MapboxCoreNavigation was resolved to 1.3.0, which depends on
      MapboxDirections (~> 1.2.0) was resolved to 1.2.0, which depends on
        Polyline (~> 5.0)

    MapboxStatic.swift was resolved to 0.11.0, which depends on
      Polyline (~> 4.2)

```

### Expected behavior

Installation successful

### Actual behavior

error

### Is this a one-time issue or a repeatable issue?

repeatable

**Comment by rgmyyw** [comment]
Can you provide the latest version? I expect it to be the latest, not concerned about stability. MapboxSearch MapboxCommon MapboxMaps MapboxCoreNavigation, Because I have used it all

**Comment by volkdmitri** [comment]
Thank you for bringing this issue to our attention! CocoaPods support is indeed on our roadmap for Mapbox Navigation version 3. Unfortunately, we do not have a specific timeline for its implementation at this moment. We will keep you updated with any new information as soon as it becomes available.

**Comment by YoussefHenna** [comment]
I'm trying to create a react native/expo wrapper for the navigation SDK and the lack of cocoapods is a real pain point in the process. Support for it soon will be greatly appreciated!

**Comment by fbeccaceci** [comment]
> I'm trying to create a react native/expo wrapper for the navigation SDK and the lack of cocoapods is a real pain point in the process. Support for it soon will be greatly appreciated!

I’m trying to integrate with expo as well, have you found any workaround to make it work?

**Comment by YoussefHenna** [comment]
> > I'm trying to create a react native/expo wrapper for the navigation SDK and the lack of cocoapods is a real pain point in the process. Support for it soon will be greatly appreciated!
> 
> I’m trying to integrate with expo as well, have you found any workaround to make it work?

Yeah, but it's an ugly process. See the bottom of the readme here https://github.com/YoussefHenna/expo-mapbox-navigation. But also, you should be able to use the package as is. Although I've yet to add a bunch of features, and there's a know bug with the latest version on Expo 50.

**Comment by fbeccaceci** [comment]
Also trying to integrate the .xcframeworks with cocoapod as vendored frameworks throws this error in xcode
`Failed to build module 'MapboxNavigationCore'; this SDK is not supported by the compiler (the SDK is built with 'Apple Swift version 5.9.2 (swiftlang-5.9.2.2.56 clang-1500.1.0.2.5)', while this compiler is 'Apple Swift version 5.10 (swiftlang-5.10.0.13 clang-1500.3.9.4)'). Please select a toolchain which matches the SDK.`

**Comment by jmysliv** [comment]
I was also trying to integrate with Expo. I ended up using different SDK versions per platform:
* v3 for Android (Maps v11 )
* v2 for iOS (Maps v10), as v2 supports cocoapods.
 
 It works, however, I would like to align those versions as soon as Mapbox adds support for cocoapods in v3.

**Comment by fbeccaceci** [comment]
> I was also trying to integrate with Expo. I ended up using different SDK versions per platform:
> 
> * v3 for Android (Maps v11 )
> * v2 for iOS (Maps v10), as v2 supports cocoapods.
> 
> It works, however, I would like to align those versions as soon as Mapbox adds support for cocoapods in v3.

I got it working with expo and v3.3.0, you gotta build the frameworks from source
There is some info and some replies of mine explaining the process here https://github.com/mapbox/mapbox-navigation-ios/issues/4703

**Comment by YoussefHenna** [comment]
@kried Any updates on when we can expect this? It's blocking proper react-native/expo integrations and development.

**Comment by 6ixNugget** [comment]
Hi there, just checking in to see if this is still on the active roadmap

**Comment by volkdmitri** [comment]
Hi all! We previously shared that CocoaPods support was on our roadmap for Mapbox Navigation SDK v3. However, following the recent [announcement](https://blog.cocoapods.org/CocoaPods-Support-Plans/) that CocoaPods is now in maintenance mode, we have reassessed our plans and decided not to move forward with CocoaPods support for v3 for now.

We understand that this may be disappointing for some users, and we appreciate your feedback. Swift Package Manager remains the primary integration method for v3, and we encourage developers to transition to it.

Thank you for your understanding, and we appreciate your feedback!

**Comment by YaoJuan** [comment]
> I'm trying to create a react native/expo wrapper for the navigation SDK and the lack of cocoapods is a real pain point in the process. Support for it soon will be greatly appreciated!

flutter, same situation!

**Comment by FernandezEnrique** [comment]
> native/expo wrapper for

Same situation here!!
